addons/sale_timesheet/__init__.py

```diff
@@ -4,3 +4,11 @@
 from . import models
 from . import wizard
 from . import report
+
+from odoo import api, SUPERUSER_ID
+
+
+def _sale_timesheet_post_init(cr, registry):
+    env = api.Environment(cr, SUPERUSER_ID, {})
+    settings = env['res.config.settings'].create({'group_uom': True})
```

Comment:
Don't create res.config on init just for that. It doesn't a shitload of other stuff. 

Add the group in the implied_ids of base.group_user instead (as done in set_values in res.config)

Reply:
And it can be done in xml on the group directly. No need of a post_init_hook ;)

Reply:
👍